src/stringify.ts

```diff
@@ -0,0 +1,34 @@
+import type { SomeSchema } from './types';
+
+export const schemaToString = (schema: SomeSchema<any>): string => {
+  const modifiers = getModifiers(schema);
+  return unionToPrint([schema.toString(), ...modifiers]);
+};
+
+export const typeToPrint = (str: string) => str;
+export const objectToPrint = (str: string) => '{' + str + '}';
+export const quote = (str: string) => (/\s/.test(str) ? `"${str}"` : str);
+
+const unionToPrint = (arr: readonly string[]): string => {
+  const str = arr.join(' | ');
+
+  if (arr.length > 1) {
+    return `(${str})`;
+  }
+  return str;
+};
+
+const getModifiers = (v: SomeSchema<any>): readonly string[] => {
+  const modifiers = [v.__modifiers.optional && 'undefined', v.__modifiers.nullable && 'null'];
+  return modifiers.filter((m): m is string => Boolean(m));
+};
```

Comment:
Won't we include `minLength`?

Reply:
For what?

Reply:
Okay, you're right, but this function name confused me.